transfers/reingest.py

```diff
@@ -0,0 +1,509 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Reingest AIPS Automatically.
+
+Created for CCArch to reingest compressed AIPs using an alternative
+Archivematica ProcessingMCP.xml file.
+
+A work in progress, with some improvements that can be made to long-running
+processes like this over time.
+"""
+
```

Comment:
You need `from __future__ import print_function` if you're going to use `print()` and if we're targetting python 2 and 3 (which we are, I believe).